ClinGen/clincoded
# #2108: Terms of Use Page

**Steps to Test #2108**

1. Click "Help" drop down at the top of the page.
2. Click "Interface Terms of Use"
3. Expect to be linked the the Terms of Use page
4. Check text to make sure grammar is correct and links work correctly.

Repeat steps while logged in and not logged in.
